ciao-launcher/main.go

```diff
@@ -93,6 +100,8 @@ func init() {
 	flag.BoolVar(&hardReset, "hard-reset", false, "Kill and delete all instances, reset networking and exit")
 	flag.BoolVar(&simulate, "simulation", false, "Launcher simulation")
 	flag.StringVar(&cephID, "ceph_id", "", "ceph client id")
+	flag.BoolVar(&prepare, "osprepare", false, "Install dependencies")
+	flag.StringVar(&roles, "roles", "agent", "Roles for which dependencies are to be installed")
```

Comment:
Suggestion: As --roles is only used for --osprepare you could make the latter a string variable and remove the duplicate

Reply:
Good idea.  I can't see any reason not to do this.   I'll make the change.

Reply:
We discussed this offline and decided not to make the change in the end as the difference in behaviour between launcher and the other services might be a little confusing.